docs/create-basic-receiver-app.md

```diff
@@ -0,0 +1,54 @@
+# Create a Basic Receiver App
+
+The following is the main structure of a basic receiver app, which has no customization:
+
+1.  A script element to load the Google Cast Receiver Framework.
+
+
+```html
+<script src="//www.gstatic.com/cast/sdk/libs/caf_receiver/v3/cast_receiver_framework.js"></script>
+```
+
+2.  A script element to load the Kaltura Receiver Player SDK.
+
+
+```html
+<script src="//www.kaltura.com/p/{YOUR_PARTNER_ID}/sp/{YOUR_PARTNER_ID}00/embedPlaykitJs/uiconf_id/{UI_CONF_ID}/partner_id/{YOUR_PARTNER_ID}"></script>
```

Comment:
please change to cdnapisec instead of www

Reply:
remove sp/{YOUR_PARTNER_ID}00 + remove partner_id/{YOUR_PARTNER_ID} we dont need them

Reply:
done